app/presenters/tslr_claims_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class TslrClaimsPresenter < BasePresenter
+  attr_accessor :decorated_collection
+
+  def initialize(claims)
+    self.decorated_collection = claims.map { |v| TslrClaimPresenter.new(v) }
```

Comment:
mapping here means that _all_ the records will be loaded into memory, and then all of them will get a presenter, which could leave us open to a DOS attack if someone spams us with lots of submissions - memory would be eaten up without the garbage collector able to clean anything up. Instead we should assign the collection and then we can use `find_each` when looping through at the point of generation to load things in batches and ensure the garbage collector can do its work.